src/Runner.Worker/JobExtension.cs

```diff
@@ -17,6 +17,7 @@
 using GitHub.Runner.Common.Util;
 using GitHub.Runner.Sdk;
 using GitHub.Services.Common;
+using Newtonsoft.Json;
```

Comment:
curious is this used?

Reply:
Apologies, it was used originally before I migrated the logic to `SnapshotOperationProvider` and I forgot to remove the reference.